linter/linter_test.go

```diff
@@ -372,6 +373,86 @@ sub vcl_recv BOOL {
 }`
 		assertError(t, input)
 	})
+
+	t.Run("Subroutines can be reused in multiple vcl state functions", func(t *testing.T) {
+		input := `
+//@recv, log
```

Comment:
Does this PR have backward compatibility? If not so, we should change this test with `@scope: recv, log`.

Reply:
yeah it also maintains backwards compatibility, so both `@scope: recv, log` are supported and also `@recv, log`

Reply:
We can remove the latter in a v2 version of falco if we start having more breaking changes that we want to make